src/v2/Apps/Partner/Routes/Overview/index.tsx

```diff
@@ -37,14 +37,17 @@ const Overview: React.FC<OverviewProps> = ({ partner }) => {
       {hasArticles && (
         <ArticlesRailFragmentContainer partnerSlug={slug} articles={articles} />
       )}
-    </Box>
+    </>
+  ) : (
+    <>Overview Non-subscribing</>
```

Comment:
This is a placeholder - right? Would be changed before any releases?

Reply:
Yes, it will be removed when we start page implementing.